src/ActionModal/components/ActionModal.vue

```diff
@@ -247,7 +247,9 @@
                 type="primary"
                 value="Next"
                 :disabled="
-                  disabled || (step === feeStep && $v.invoiceTotal.$invalid)
+                  disabled ||
+                    (step === feeStep && $v.invoiceTotal.$invalid) ||
+                    (step === feeStep && !estimatedFee)
```

Comment:
doesn't work. if estimated fee is 0 this will always block.

Reply:
```suggestion
                    (step === feeStep && !gasEstimateLoaded)
```

Reply:
oh true!